app/scripts/utils/path.js

```diff
@@ -1,26 +1,28 @@
-function Path (protocol, name, path) {
-  this.protocol = protocol
-  this.name = name
-  this.path = path
+export default class Path {
```

Comment:
yes this is ES2015 class syntax: http://www.2ality.com/2015/02/es6-classes-final.html